pkg/collector/corechecks/containers/containerd/events.go

```diff
@@ -296,3 +308,29 @@ func (s *subscriber) Flush(timestamp int64) []containerdEvent {
 	s.Events = nil
 	return ev
 }
+
+var topicsToEventType = map[string]string{
+	"/containers/create": "create",
+	"/containers/delete": "destroy",
+	"/containers/update": "update",
+	"/tasks/create":      "create",
+	"/tasks/delete":      "destroy",
+	"/tasks/exit":        "die",
+	"/tasks/oom":         "oom",
```

Comment:
I don't see the same logic than in the docker check to create an "Alert" event for `kill` and `oom`.

Also here it is "die". should we use also "kill" ?